DMD2_Timer.cpp

```diff
@@ -159,6 +166,50 @@ void BaseDMD::end()
   scanDisplay();
 }
 
+#elif defined(ESP32)
+hw_timer_t *timer = NULL;
```

Comment:
timer should be static to use two instances of DMD in the same time
```suggestion
static hw_timer_t *timer = NULL;
```